### ISSUE TITLE
TextField component error message not being announced in EDGE, IE and FireFox

### ISSUE THREAD

**Issue by b-priyank** [OfficeDev/office-ui-fabric-react#1272]
### Bug Report

- Version: office-ui-fabric-react 1.14.2
- Browsers and OS: EDGE, IE, FireFox in Widows10

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)
No

Requested priority: (Blocking, High, Normal, Low) 
Normal
#### Describe the issue:

TextField errorMessage (live-region) is being announced only in Chrome. It is not being acknowledged in EDGE, IE and FireFox

#### Actual behavior:

Announced nothing when errorMessage is displayed while editing TextField.

#### Expected behavior:

It should announce errorMessage whenever displayed while editing TextField.

**Comment by sulabh-msft** [comment]
@mdahamiwal Is this issues resolved now?

**Comment by mdahamiwal** [comment]
Yes, this should be resolved now.
